packages/in_app_purchase/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -295,7 +348,8 @@ public void onBillingSetupFinished(BillingResult billingResult) {
               return;
             }
             alreadyFinished = true;
-            // Consider the fact that we've finished a success, leave it to the Dart side to validate the responseCode.
+            // Consider the fact that we've finished a success, leave it to the Dart side to
```

Comment:
it looks like there's some formatting issue?

Reply:
Added a line break since the comment was long (autoformatted by 'google-java-format')